silkworm/silkrpc/ethdb/listener_cbor_log.cpp

```diff
@@ -0,0 +1,85 @@
+/*
```

Comment:
This file should be renamed as `log_cbor_listener.cpp`

Reply:
file will be removed